### DISCUSSION TITLE
[Frigate+ Question]: Will images uploaded to Frigate+ for one camera impact the rest of the cameras

### DISCUSSION THREAD

**Discussion by heisenberg2980** [blakeblackshear/frigate#12311] [Ask A Question]
### What is your question:

I would like to understand how the Frigate+ model is generated and if the images uploaded are treated equally irrespective of the camera they correspond to. I am thinking about the possibility of having cameras from different qualities or positions, could the images uploaded from one camera negatively impact the detection on the other cameras, or is the model somehow different per camera and aware of the particular camera to apply only the related pictures used to train the model from that specific camera? Or is the model more likely to relate the pictures uploaded from one specific camera as they will be more similar to the image it is analysing so this won´t be a problem?

**Comment by NickM-27** [comment]
nothing is done for the specific cameras, but also I don't think it really causes any issues. For example I have a wide angle panoramic camera, doorbell camera, and many normal 16:9 roofline mounted cameras and this difference has not caused any issues
